lib/streamlit/app_session.py

```diff
@@ -96,6 +97,15 @@ def __init__(
         local_sources_watcher: LocalSourcesWatcher
             The file watcher that lets the session know local files have changed.
 
+        user_info: Dict
+            Dict that contains information about current user.
+            For now it is just an email.
```

Comment:
Maybe we can find better wording here?

Reply:
The wording here seems okay to me. Maybe it'd be good to tweak it very slightly to something like

```
user_info: Dict
    A dict that contains information about the current user. For now, it only contains
    the user's email address.
    <example dict>
    Information about the current user is optionally provided when a websocket connection
    is initialized via the "X-Streamlit-User" header.